test/e2e/cypress/integration/account-panel.spec.ts

```diff
@@ -0,0 +1,28 @@
+describe('Account panel', () => {
+  const firstName = 'Name';
+  const lastName = 'Lastname';
+  const email = 'name@intx.com';
+
+    beforeEach(() => {
+      cy.clearLocalStorage();
+      cy.login();
+      cy.visit('/preferences?tab=account');
+    });
+
+    it('Should edit name and lastname', () => {
+      cy.get('button').contains('Edit').click();
+      cy.get('input[name=firstName]').type(firstName);
+      cy.get('input[name=lastName]').type(lastName);
+      cy.get('button').contains('Save').click();
+      cy.contains(firstName);
```

Comment:
Can you please check if this `contains` awaits the change? or does it check the already typed result.

Reply:
Yes, wait for the change